src/main/java/io/phasetwo/service/auth/storage/datastore/PhaseTwoDatastoreProvider.java

```diff
@@ -0,0 +1,19 @@
+package io.phasetwo.service.auth.storage.datastore;
+
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.storage.ExportImportManager;
+import org.keycloak.storage.datastore.DefaultDatastoreProvider;
+
+public class PhaseTwoDatastoreProvider extends DefaultDatastoreProvider {
```

Comment:
We should use naming like `KeycloakOrgsDatastoreProvider` rather than Phase Two

Reply:
Done